ansible/README.md

```diff
@@ -175,6 +181,8 @@ credentials are quay_username and quay_password, which are created by make
 from the environment variables `QUAY_RHACS_ENG_RO_USERNAME` and `QUAY_RHACS_ENG_RO_PASSWORD`
 to match CI variables. If you are using IBM Cloud to create IBM Z RHEL instances, you will also need to specifiy enviroment variables `REDHAT_USERNAME` and `REDHAT_PASSWORD` to register your RHEL system.
 
+If you are using IBM Cloud to create IBM Z and Power RHEL instances, you will also need to specifiy enviroment variables `REDHAT_USERNAME` and `REDHAT_PASSWORD` to register your RHEL system.
```

Comment:
This is duplicating the line above - if we just remove `Z` from line 182, then it covers both cases

Reply:
done